test/system/030-run.bats

```diff
@@ -1382,4 +1382,14 @@ search               | $IMAGE           |
     run_podman container rm $cname
 }
 
+# https://issues.redhat.com/browse/RHEL-14469
+@test "podman run - /run must not be world-writable in systemd containers" {
+    run_podman run -d --rm $SYSTEMD_IMAGE /usr/sbin/init
+    cid=$output
+    run_podman exec -it $cid stat -c '%a' /run
```

Comment:
I put a comment there. I did so for a reason: if the test ever fails, and someone goes looking into the code, this comment will make it possible for them to understand what's going on. Without that comment, this is just magic.
```bash
    # runc has always been 755; crun < 1.11 was 777
```

Reply:
Make sence.

Reply:
Done